src/main/kotlin/no/nav/syfo/client/DokArkivClient.kt

```diff
@@ -90,7 +90,7 @@ class DokArkivClient(
     suspend fun ferdigstillJournalpost(
         journalpostId: String,
         msgId: String,
-        veileder: Veileder,
+        veileder: Veileder, // TODO: Denne bør brukes? Hvor gjør vi det?
```

Comment:
Brukes den ikke bare ved ferdigstilling av oppgaven, ikke ved ferdigstilling av journalposten?

Reply:
Jeg synes forresten ikke vi bør ha retry på det som kalles fra frontend, for hvis den prøver igjen og igjen timer det antageligvis bare ut mot frontend uansett. Bedre å la det feile med en gang så bruker kan prøve på nytt fremfor at bruker må vente leeenge og så timer det ut for frontend, og så må man kanskje, kanskje ikke gjøre kallet på nytt uansett.